src/vcpkg/vcpkgpaths.cpp

```diff
@@ -703,6 +723,38 @@ namespace vcpkg
         }
     }
 
+    /// <summary>
+    /// This is used to translate relative paths in the configuration file
+    /// to be relative to the file instead of the working directory
+    /// </summary>
+    void VcpkgPaths::translateRelativePaths()
```

Comment:
```suggestion
    void VcpkgPaths::translate_relative_paths()
```

snake_case_for_the_win